### ISSUE TITLE
Asset path is not being set correctly on startup

### ISSUE THREAD

**Issue by Bonkles** [facebook/Rapid#1034]
### Description

This isn't a problem for most normal dev builds and even testing the sandbox builds in their upload dirs on our AWS cloud environ, but once we start shuffling URLs around and moving things to test urls, assets start failing to load because the code is looking in `https://rapideditor.org/locales/index.min.json`, which does not exist. We need to ensure that the `asset path` is set correctly so that this URL works for release builds getting copied different places. 

### Version

2.1

### What browser are you seeing the problem on? What version are you running?

All

### The OS you're using

n/a

### Steps to reproduce

Navigate to any test URL, such as 

https://rapideditor.org/rapid/b1ab600-16.x-/index.html#background=EsriWorldImagery&datasets=fbRoads,msBuildings&disable_features=boundaries&map=2.00/-48.0/122.5

And observe that it works fine. 

Now, using the AWS console, migrate its base index.html file to another directory and rename it. Things SHOULD still work, but they do not. Instead you will get many console messages complaining about missing resources: 

```

Failed to load resource: the server responded with a status of 404 ()
LocalizationSystem.js:202 FetchError: 404 
    at Ti (fetch_response.js:40:11)
    at async Promise.all (index 0)
(anonymous) @ LocalizationSystem.js:202
LocalizationSystem.js:380 Missing en translation: loading_auth
tHtml @ LocalizationSystem.js:420
(anonymous) @ UiSystem.js:92
LocalizationSystem.js:380 Missing en translation: area_fill.wireframe.key
t @ LocalizationSystem.js:407
(anonymous) @ MapSystem.js:111
LocalizationSystem.js:380 Missing en translation: map_data.highlight_edits.key
t @ LocalizationSystem.js:407
(anonymous) @ MapSystem.js:112
LocalizationSystem.js:380 Missing en translation: rapid_feature_toggle.fbRoads.label
t @ LocalizationSystem.js:407
(anonymous) @ RapidSystem.js:89
LocalizationSystem.js:380 Missing en translation: rapid_feature_toggle.fbRoads.license_markdown
t @ LocalizationSystem.js:407
(anonymous) @ RapidSystem.js:90
LocalizationSystem.js:380 Missing en translation: rapid_feature_toggle.msBuildings.label
t @ LocalizationSystem.js:407
(anonymous) @ RapidSystem.js:101
LocalizationSystem.js:380 Missing en translation: rapid_feature_toggle.msBuildings.license_markdown
t @ LocalizationSystem.js:407
(anonymous) @ RapidSystem.js:102
locales/index.min.json:1     Failed to load resource: the server responded with a status of 404 ()
data/imagery.min.json:1     Failed to load resource: the server responded with a status of 404 ()
fetch_response.js:40 Uncaught (in promise) FetchError: 404 
    at Ti (fetch_response.js:40:11)
    at async Promise.all (index 3)
```

### The browser URL at the time you encountered the bug

n/a

**Comment by Bonkles** [comment]
This turned out to be an issue in our aws deploy script, which find-and-replaces all the instances of asset path usage in the main index.html and changes them around. Since the old `context.assetPath()` setter is now a property (`Context.assetPath = ''`) the deploy script just needed a small tweak to do the right thing.

**Comment by Bonkles** [comment]
Fixed in `b65d526f5`.
